File: plugins/OrchidForms/src/main/java/com/eden/orchid/presentations/FormsResourceSource.java
package com.eden.orchid.presentations;

import com.eden.orchid.api.OrchidContext;
import com.eden.orchid.api.resources.resourceSource.PluginResourceSource;

import javax.inject.Inject;
import javax.inject.Singleton;

@Singleton
public class FormsResourceSource extends PluginResourceSource {

    @Inject
    public FormsResourceSource(OrchidContext context) {
        super(context, 20);
    }

}

File: plugins/OrchidForms/src/main/kotlin/com/eden/orchid/forms/FormsResourceSource.kt
package com.eden.orchid.forms

import com.eden.orchid.api.OrchidContext
import com.eden.orchid.api.resources.resourceSource.PluginResourceSource

import javax.inject.Inject
import javax.inject.Singleton

@Singleton
class FormsResourceSource @Inject
constructor(context: OrchidContext) : PluginResourceSource(context, 20)
